src/web-ui/package.json

```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "@optimizely/optimizely-sdk": "4.0.0",
     "amplitude-js": "^5.8.0",
-    "aws-amplify": "^3.0.23",
+    "aws-amplify": "^3.0.24",
```

Comment:
This on its own actually resolves all the dependency issues according `npm audit`. This is the case even with all of the dependencies removed.